cmd/nodeagent/nodeagentmanager_test.go

```diff
@@ -41,7 +41,7 @@ var _ = Describe("nodeagent tests", func() {
 	It("heartbeat with empty body should return error", func() {
 		req := httptest.NewRequest(http.MethodPost, "/v1/sessionHosts/sessionHostID", nil)
 		w := httptest.NewRecorder()
-		n := NewNodeAgentManager(newDynamicInterface(), testNodeName, false, false, time.Now)
+		n := NewNodeAgentManager(newDynamicInterface(), testNodeName, false, false, time.Now, true)
```

Comment:
do some tests need the checker to run while some others don't?

Reply:
There are 5 heartbeat tests, and in 3 of these i run the checker manually, so i turned the loop off in these cases.